PWGHF/D2H/Tasks/taskLc.cxx

```diff
@@ -704,57 +548,263 @@ struct HfTaskLc {
     }
   }
 
-  void processMcStd(CollisionsMc const& collision,
+  /// Fill MC histograms at reconstruction level
+  /// \param collision is collision
+  /// \param candidates is candidates
+  template <bool fillMl, typename CollType, typename CandType, typename TrackType>
+  void fillDataHistosAndSparse(CollType collision, CandType const& candidates, TrackType const& tracks)
+  {
+
+    auto thisCollId = collision.globalIndex();
+    auto groupedLcCandidates = candidates.sliceBy(candLcPerCollision, thisCollId);
+
+    int nTracks = 0;
+    if (collision.numContrib() > 1) {
+      for (const auto& track : tracks) {
+        if (std::abs(track.eta()) > 4.0) {
+          continue;
+        }
+        if (std::abs(track.dcaXY()) > 0.0025 || std::abs(track.dcaZ()) > 0.0025) {
+          continue;
```

Comment:
No magic numbers. These selections should be defined in constants or configurables.

Reply:
> No magic numbers. These selections should be defined in constants or configurables.

yes, I agree, but I think this be implemmed by someone long time ago. Here I just remove it and fill the number contributor in the histogram. Someone can implement more detail for the multiplicity study later for this

Reply:
Don't remove it. Someone added the histogram for a reason.